### PR TITLE
Adding link to FINOS Privacy Policy

### PR DESCRIPTION
**Pull request by maoo**
Check preview on http://blog.session.it/alloy/docs/contribute .

### CODE REVIEW

File: website/core/Footer.js [hunk 1]
```diff
@@ -29,9 +29,14 @@ class Footer extends React.Component {
           </a>
         </section>
         <section className="finos finosBanner">
-          <a href="https://www.finos.org">
+          <div>
             <img id="finosicon" src={`https://alloy.finos.org/img/finos_wordmark.svg`} height='75px' alt="FINOS" title="FINOS"/>
-          </a>
+            <div>
+              <a class="nav-home" href="/">Alloy Documentation</a><br/>
+              <a class="nav-home" href="https://www.finos.org" target="_blank">FINOS website</a><br/>
```
Comment (gs-bracej) [inline website/core/Footer.js]: Do we need these first two links? The FINOS logo is already a link to the FINOS website. What if we just have Privacy Policy?

Comment (brooklynrob) [reply website/core/Footer.js]: Agreed. I shared the same feedback with @maoo last week. I think it should just be

- FINOS icon itself is a link to the FINOS website
- link to the privacy policy
- remove link to "Alloy Documentation (which is a circular anyway -- it's a link the site the site you're already on)"

**Comment by maoo** [comment]
Sounds great @gs-bracej . I added those just to build a sort of link menu, for visual purposes. Feel free to get rid of them, what matters is the FINOS Privacy Policy link. TY!

File: website/core/Footer.js (hunk 1 above)
Comment (brooklynrob) [reply website/core/Footer.js]: @maoo @gs-bracej I'd suggest we make the changes above before this PR is approved.

**Comment by maoo** [comment]
Closing this PR, issue is being worked by GS, see #80

**Pull request closed without merging**